src/components/Projects/ProjectSheetStatusPill.tsx

```diff
@@ -31,10 +31,6 @@ export default function ProjectSheetStatusPill({ project, hero = false }: Props)
 
   const pillData = getPillTextData(status, created_at, funding)
```

Comment:
If we are not showing days for finished status then you can remove that from `getPillTextData`

Reply:
it is the same pill used for the ongoing status that shows the days